RPIPs/RPIP-43.md

```diff
@@ -72,14 +72,15 @@ Node operators can manage the set of validators in their megapool:
 - The oDAO SHALL be able to increase `debt` to apply penalties (see [RPIP-42](./RPIP-42.md/#penalizable-offenses)) by majority vote
   - If this causes the megapool to meet the criteria for a non-node operator initiated exit (see [RPIP-44](RPIP-44.md)), it is RECOMMENDED that an exit is initiated by the same transaction 
 - There SHALL be a function to pay off `debt` with ETH. The ETH SHALL be sent to the rETH contract and the `debt` SHALL be reduced by the amount.
+^ does the smoothing pool need to be accounted for here if they are opted into it? ^
 
 
 ### Funds Management
 
 In the following, "capital" refers to funds supplied for staking by either the
 node operator or pool stakers, including node operator staked RPL. "Rewards"
 refers to additional funds generated or obtained via staking, including
-consensus rewards, execution rewards, and RPL inflation. The split of rewards
+consensus rewards, execution rewards, RPL issuance, and `voter_share` rewards. The split of rewards
```

Comment:
I don't think we need to add voter share. This is consensus and execution rewards before then being split into shares per UARS (next sentence).

Reply:
I can take or leave issuance vs inflation

Reply:
Ok, I'll update to just:
> consensus rewards, execution rewards, and RPL issuance. The split of rewards

Reply:
:rocket: